pnparser/templates/common.py

```diff
@@ -0,0 +1,23 @@
+# static
+ONLY_INCLUDE: str = "<onlyinclude>"
+BOX_START: str = "{{Box|Start}}\n"
```

Comment:
can you lowercase the argument - `{{Box|start}}` etc (they're supposed to be lowercase, the only reason uppercase works is because we sanitize user input haha)

Reply:
Done.